_layouts/address.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
+
+{% include head.html %}
+
+<body>
+  {% include header-extended.html %}
+
+  <main id="main-content">
+    <div class="grid-container flex-align-center margin-y-5">
+      <div class="display-flex flex-column flex-align-center">
+        <form class="usa-form usa-form--large">
+          <fieldset class="usa-fieldset">
+            <legend class="usa-legend">
+              <h1 class="margin-y-1">
+                {{ page.title }}
+              </h1>
+            </legend>
+            <p class="margin-y-1">
+              Required fields are marked with an asterisk (<abbr title="required" class="usa-hint usa-hint--required">*</abbr>).
+            </p>
+            <h2 class="margin-top-6">Contact Details</h2>
+            {% for label in page.contact-labels %}
+              <label class="usa-label" for="{{label.input-name}}">
+                {{ label.label }}
+                {% if label.required %}
+                  <abbr title="required" class="usa-hint usa-hint--required">*</abbr>
+                {% else %}
+                  <abbr title="optional"></abbr>
+                {% endif %}              
+              </label>
+              <input 
+              class="usa-input" 
+              id="{{label.input-name}}" 
+              name="{{label.input-name}}" 
+              {% if label.required %} required {% endif %}
+            >
+            {% endfor %}
+
+            <div class="usa-checkbox margin-top-2">
+              <input class="usa-checkbox__input" id="receive-email-updates" type="checkbox" name="receive-email-updates" value="receive-email-updates">
+              <label class="usa-checkbox__label" for="receive-email-updates">Yes, I would like to receive updates on my status</label>
```

Comment:
I think this is fine for this iteration, but since you've come with a way to separate all the other content, it might be nice to figure out a solution for these one-off checkboxes.

Reply:
👍